test/gui/shared/scripts/helpers/api/Provisioning.py

```diff
@@ -51,3 +32,22 @@ def setup_app(app_name, action):
         disable_app(app_name)
     else:
         raise Exception("Unknown action: " + action)
+
+
+def create_group(group_name):
+    global created_groups
+    if get_config('ocis'):
+        group_info = ocis.create_group(group_name)
+    else:
+        group_info = oc.create_group(group_name)
+    created_groups[group_name] = group_info
+
+
+def delete_groups():
```

Comment:
can we renamed it like this so that we know this functions only tries to delete the groups created during the test
```suggestion
def delete_created_groups():
```

Reply:
what about writing delete_group directly in delete_created_groupsps, without creating separate func
```suggestion
def delete_created_groups():
    global created_groups
    for group_name, group_info in list(created_groups.items()):
        if get_config('ocis'):
            url = url_join(ocis.get_ocis_url(), 'groups', group_id)
        else:
           url = url_join(oc.get_ocs_url(), 'groups', group_id)
        response = request.delete(url)
        request.assertHttpStatus(response, 200)
        del created_groups[group_name]
```

Reply:
let's have delete_group function for each server in own file. existing appraoch is fine